src/test/resources/fixtures/govuk-frontend/patched-fixtures/header-navigation-item-with-text-without-link/output.txt

```diff
@@ -33,7 +33,7 @@
       Service Name
     </a>
     <nav aria-label="Menu" class="govuk-header__navigation ">
-      <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide menu">Menu</button>
+      <button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide menu" hidden>Menu</button>
```

Comment:
@kyle-bowden I think it's something we need to capture in a new ticket, but I've noticed that the [govuk header has got an extra hidden attribute here](https://github.com/alphagov/govuk-frontend/commit/14e1661eed31160b9cabf76d0c5fba1052525114), so we probably need to update hmrc-header https://github.com/hmrc/hmrc-frontend/blob/main/src/components/header/template.njk#L63

Reply:
The work for this change has been done in a separate [PR](https://github.com/hmrc/hmrc-frontend/pull/242/files) and the related ticket is [here](https://jira.tools.tax.service.gov.uk/browse/PLATUI-1951)